src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -566,6 +566,15 @@ private void validatePmp(Pmp pmp, int impIndex) throws ValidationException {
 
     private void validateBanner(Banner banner, int impIndex) throws ValidationException {
         if (banner != null && banner.getFormat() != null) {
+            final boolean hasWidth = hasValue(banner.getW());
+            final boolean hasHeight = hasValue(banner.getH());
+            final boolean hasSizes = hasWidth && hasHeight;
+
+            if (!hasSizes && banner.getFormat().isEmpty()) {
```

Comment:
As far as I understand this check will not be performed if `banner.getFormat() == null`, but it should